Solutions/Lookout/Parsers/LookoutEvents.txt

```diff
@@ -23,7 +24,8 @@ let LookoutEvents_view  = view () {
 		DetailsAttributeChanges=column_ifexists('details_attributeChanges_s', ''),
 		Type=column_ifexists('type_s', ''),
 		ID=column_ifexists('id_s', ''),
-		EventTime=column_ifexists('eventTime_t', ''),
+        EventStartTime=column_ifexists('eventTime_t', ''),
+        TimeGenerated=column_ifexists('eventTime_t', ''),
```

Comment:
is it intensional change? Generally all tables in a Log Analytics workspace have a column called TimeGenerated which is the time that the record was created.